src/CommonITILObject.php

```diff
@@ -8720,17 +8722,50 @@ public static function getKanbanColumns($ID, $column_field = null, $column_ids =
             $columns[$item[$column_field]]['items'][] = $card;
         }
 
-       // If no specific columns were asked for, drop empty columns.
-       // If specific columns were asked for, such as when loading a user's Kanban view, we must preserve them.
-       // We always preserve the 'No Status' column.
+        // If no specific columns were asked for, drop empty columns.
+        // If specific columns were asked for, such as when loading a user's Kanban view, we must preserve them.
+        // We always preserve the 'No Status' column.
```

Comment:
Maybe this comment should be moved in first `if` block inside the `foreach` loop, as it is irrelevant for the `else if` block.